rln/tests/poseidon_tree.rs

```diff
@@ -91,47 +91,31 @@ mod pmtree_test {
     use rln::poseidon_hash::poseidon_hash;
     use rln::poseidon_tree::PoseidonHash;
     use rln::protocol::hash_to_field;
-    use rln::utils::{bytes_le_to_fr, fr_to_bytes_le, str_to_fr};
+    use rln::utils::str_to_fr;
     use sled::Db as Sled;
-    use std::collections::HashMap;
     use std::fs;
-    use std::path::Path;
+    use std::{collections::HashMap, path::PathBuf};
     use utils::{FullMerkleTree, OptimalMerkleTree};
 
-    // The pmtree Hasher trait used by pmtree Merkle tree
-    impl pmtree::Hasher for PoseidonHash {
-        type Fr = Fr;
-
-        fn default_leaf() -> Self::Fr {
-            Fr::from(0)
-        }
-
-        fn serialize(value: Self::Fr) -> Value {
-            fr_to_bytes_le(&value)
-        }
-
-        fn deserialize(value: Value) -> Self::Fr {
-            let (fr, _) = bytes_le_to_fr(&value);
-            fr
-        }
-
-        fn hash(inputs: &[Self::Fr]) -> Self::Fr {
-            poseidon_hash(inputs)
-        }
-    }
-
     // pmtree supports in-memory and on-disk databases (Database trait) for storing the Merkle tree state
 
     // We implement Database for hashmaps, an in-memory database
     struct MemoryDB(HashMap<DBKey, Value>);
 
+    #[derive(Default)]
+    struct MemoryDBConfig {
+        dbpath: PathBuf,
```

Comment:
This field can be empty for now

Reply:
Done here: [3ff0735](https://github.com/vacp2p/zerokit/pull/125/commits/3ff0735af530cfdc02ec7c4c9bd48ed790842d51)